src/core/router/net_db/impl.cc

```diff
@@ -951,13 +951,14 @@ std::shared_ptr<const RouterInfo> NetDb::GetClosestFloodfill(
   min_metric.SetMax();
   std::unique_lock<std::mutex> l(m_FloodfillsMutex);
   for (auto it : m_Floodfills) {
```

Comment:
May as well `const auto&` while you're here.

Reply:
Will do.